lib/unicode_plot/braille_canvas.rb

```diff
@@ -46,7 +46,7 @@ def pixel!(pixel_x, pixel_y, color)
       index = index_at(char_x - 1, char_y - 1)
       if index
         @grid[index] = (@grid[index].ord | BRAILLE_SIGNS[char_x_off - 1][char_y_off - 1]).chr(Encoding::UTF_8)
-        @colors[index] |= COLOR_ENCODE[color]
+        @colors[index] |= COLOR_ENCODE.fetch(color, color)
```

Comment:
The idea is that if color is a symbol (e.g. `:blue`), then it will fetch a number.  If color is a number it will fail to find in the Hash, and just pass the color through.